### PR TITLE
fix: add support for refreshing (clearing) cached services

### LINKED ISSUE #410: Ack Per Session Action sometimes fails
Scenario: notices are displayed in notification-banner with Acknowledge (Per Session) Action.

Sometimes, when the user acknowledges that notices after logging in, the notices are displayed again on second page refresh.

### PR DESCRIPTION
**Pull request by bjagg**
Resolves #410

### CODE REVIEW

File: .java-version [hunk 1]
```diff
@@ -0,0 +1 @@
+1.8
```
Comment (ChristianMurphy) [inline .java-version:1]: Nice! I'm all for tracking versions in a file based approach.
Which toolchain uses `.java-version` files?

I'm personally more familiar with [sdkman](https://sdkman.io), which uses [`.sdkmanrc` files](https://sdkman.io/usage#env).
I'm open to any solution that makes managing the versions easier :smile:

File: notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java [hunk 2]
```diff
@@ -75,6 +75,8 @@
     public List<NotificationEntry> fetchNotifications(HttpServletRequest request) {
         final NotificationResponse response = repository.fetch(request);
         final List<NotificationEntry> rslt = notificationResponseFlattener.flatten(response);
+        logger.debug("rslt count = {}", rslt.size());
```
Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java:78]: For logs, full words often are easier to process
```suggestion
        logger.debug("result count = {}", rslt.size());
```

File: notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java [hunk 3]
```diff
@@ -75,6 +75,8 @@
...
+        logger.debug("rslt = {}", rslt);
```
Comment (ChristianMurphy) [inline notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java:79]:
```suggestion
        logger.debug("result = {}", rslt);
```

File: .java-version (hunk 1 above)
Comment (bjagg) [reply .java-version:1]: `jenv` from Homebrew IIRC

File: notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java (hunk 2 above)
Comment (bjagg) [reply notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java]: Good catch!

**Pull request merged**